lib/linter-integration.js

```diff
@@ -0,0 +1,31 @@
+'use babel';
+
+export default class LinterIntegration {
+  constructor(linterInstance) {
+    this.linter = linterInstance;
+    linterInstance.addLinter(this.provider);
+  }
+
+  get provider() {
```

Comment:
Thanks. This is the first time I've used ES6 classes, it felt a little derpy handling the object like that. I'll fix it.